README.md

```diff
@@ -198,6 +198,33 @@ npm run audit
 
 See also the [the full list of default scripts](https://github.com/rooseveltframework/roosevelt/blob/master/lib/defaults/scripts.json).
 
+Setting Up Environment Variable
+===
+
+Roosevelt's HTML Validator can be set up to be ran in the detached or attached state by default through use of the environment variable, `IS_DETACHED`.
+There are a few different ways that you can go about setting up this environment variable, depending on what OS your machine is running:
+
+Windows
+---
+1. Open up your Control Panel and navigate to System and Security, and then System.
```

Comment:
I love the effort here, you did a great job. But it is way to verbose. Explaining how to set ENV variables is out of the scope of our project. At most we could link guides in how to do set ENV variables. That renders this section kinda outdated. We could create a section outlining all ENV variables, **and**/or add the ENV variable specifically to a section talking about the validator